prefab_build.sh

```diff
@@ -0,0 +1,88 @@
+# Test that the prefab binary exists
+if hash prefab 2>/dev/null; then
+  echo "Prefab is installed"
+else
+  echo "Prefab binary not found. See https://github.com/google/prefab for install instructions"
+  exit 1
+fi
+
+# Get the version string from the source
+major=`grep "#define OBOE_VERSION_MAJOR" include/oboe/Version.h | cut -d' ' -f3`
+minor=`grep "#define OBOE_VERSION_MINOR" include/oboe/Version.h | cut -d' ' -f3`
+patch=`grep "#define OBOE_VERSION_PATCH" include/oboe/Version.h | cut -d' ' -f3`
+version=$major"."$minor"."$patch
+
+echo "Building libraries for Oboe version "$version
+./build_all_android.sh
+
+mkdir -p build/prefab
+cp -R prefab/* build/prefab
+
+ABIS=("x86" "x86_64" "arm64-v8a" "armeabi-v7a")
+
+pushd build/prefab
+
+  # Remove .DS_Store files as these will cause the prefab verification to fail
+  find . -name ".DS_Store" -delete
+
+  # Write the version number into the various metadata files
+  mv oboe-VERSION oboe-$version
+  mv oboe-VERSION.pom oboe-$version.pom
+  sed -i '' -e "s/VERSION/${version}/g" oboe-$version.pom oboe-$version/prefab/prefab.json
+
+  # Copy the headers
+  cp -R ../../include oboe-$version/prefab/modules/oboe/
+
+  # Copy the libraries
+  for abi in ${ABIS[@]}
+  do
+    echo "Copying the ${abi} library"
+    cp -v "../${abi}/liboboe.so" "oboe-${version}/prefab/modules/oboe/libs/android.${abi}/"
+  done
+
+  # Verify the prefab packages
+  for abi in ${ABIS[@]}
+  do
+
+    prefab --build-system cmake --platform android --os-version 29 \
+    --stl c++_shared --ndk-version 21 --abi ${abi} \
```

Comment:
indent

Reply:
Not sure what you mean? It is indented already using 2 spaces.

Reply:
In android style, continuation lines are usually indented by 2 levels. That would be an extra 4 spaces in this case relative to line 47.

Reply:
I getcha. Thanks for the explanation - I wasn't aware of the rule. Fixed.